.circleci/config.yml

```diff
@@ -329,7 +302,15 @@ jobs:
           # Run the test even if previous failed
           when: always
           command: |
-                make test/local/auth/e2e_pilotv2 | tee -a /go/out/tests/build-log.txt
+                make test/local/auth/e2e_pilotv2
+      - run:
+          no_output_timeout: 20m
+          when: always
+          command: |
+            # This is a fast test (<1 min), it takes more time to prepare.
+            export PATH=$GOPATH/bin:$PATH
+            make localTestEnv
+            make test/local/cloudfoundry/e2e_pilotv2
```

Comment:
I think this was how it was earlier.. but somehow, it was either failing or having resource contention issues. We could tack this into our main test job, and that would make it simple. (and it would still run in a container, as opposed to a VM)

Reply:
It's 45 seconds.. I can add it to test instead ? Maybe later ?